packages/@sanity/core/src/commands/dataset/copyDatasetCommand.js

```diff
@@ -103,33 +143,19 @@ export default {
         `Copying dataset ${chalk.green(sourceDatasetName)} to ${chalk.green(targetDatasetName)}...`
       )
 
-      if (!flags.wait) {
+      if (flags.background) {
+        /*
+         * Use doesn't want to wait for copy to finish.
+         */
+        output.print(`Copy initiated. Received job ID: ${chalk.blue(response.jobId)}`)
```

Comment:
I added this initially, but was removed due it being very technical. Thoughts, @rexxars?

Reply:
Yeah that's another good point 🙂 We could remove the job ID from this printout considering the whole `--progress` command is printed just below it.

Reply:
I figure we might as well remove it - until we have a stronger concept of the job system, this can possibly be a bit confusing right now.

Reply:
That makes sense! 🙂 Removed in 27d1a6373